### PR TITLE
Colour styling

### PR DESCRIPTION
**Pull request by SavageWilliam**
#67 Colours and beautify! 😸 

Sorry.. I almost added issue to every commit.. getting better!

### CODE REVIEW

File: css/create-ladder.css [hunk 1]
```diff
@@ -228,4 +232,44 @@ p {
...
+  margin: 0;
+}
+
+input[type=range] {
+  -webkit-appearance: none;
+  width: 100%;
+  border-radius: 8px;
+  height: 1.5rem;
+  border: 1px solid #3B3561;
+  /*background-color: #fff;*/
```
Comment (des-des) [inline css/create-ladder.css]: :fire:

File: create-ladder.js [hunk 2]
```diff
@@ -74,6 +82,20 @@
...
+  const seekslider = document.getElementById("slider");
+
+  seekslider.addEventListener('input', function() {
+    const v = this.value;
+    seekslider.style.background = "-moz-linear-gradient(left,  #3B3561 0%, #3B3561 "+ v +"%, white "+ v +"%, white 100%)";
+    seekslider.style.background = "-webkit-gradient(linear, left top, right top, color-stop(0%,#3B3561), color-stop("+ v +"%,#3B3561), color-stop("+ v +"%,white), color-stop(100%,white))";
+    seekslider.style.background = "-webkit-linear-gradient(left,  #3B3561 0%,#3B3561 "+ v +"%,white "+ v +"%,white 100%)";
+    seekslider.style.background = "-o-linear-gradient(left,  #3B3561 0%,#3B3561 "+ v +"%,white "+ v +"%,white 100%)";
+    seekslider.style.background = "-ms-linear-gradient(left,  #3B3561 0%,#3B3561 "+ v +"%,white "+ v +"%,white 100%)";
+    seekslider.style.background = "linear-gradient(to right,  #3B3561 0%,#3B3561 "+ v +"%,white "+ v +"%,white 100%)";
```
Comment (des-des) [inline create-ladder.js]: I do not this this will work, you are re-assigning `seekslider.style.background` 6 times, only the last assignment will hold..

Comment (jsms90) [inline create-ladder.js]: Isn't this how you deal with different browsers?

Comment (des-des) [inline create-ladder.js]: This is javascript, not css. the was css cascades is different to how assignment works in js.

This code is the same as the following css
```css
#slideslider {
  background: linear-gradient(to right,  #3B3561 0%,#3B3561 "+ v +"%,white "+ v +"%,white 100%);
}
```

**Pull request merged**